autoload/go/alternate.vim

```diff
@@ -0,0 +1,36 @@
+" By default use edit to switch to the new alternate buffer
+if !exists("g:go_alternate_mode")
+	let g:go_alternate_mode = "edit"
+endif
+
+" By default create the test file if it does not exist
+if !exists("g:go_create_test_file")
+	let g:go_create_test_file  = 1
```

Comment:
Ok I think we don't need this setting at all. Let's do this way. If e bang is passed it should force open the test file, if not passed it should give an error. Example:


Tries to open `foo_test.go`, if it doesn't exists it echoes an error:
```
:GoAlternate
```

Tries to open `foo_test.go`, if it doesn't exists it **creates** the `foo_test.go` file.
```
:GoAlternate!
```